lib/components/Radio/Radio.tsx

```diff
@@ -1,25 +1,59 @@
-import React, { forwardRef } from 'react';
+import React, { forwardRef, useContext } from 'react';
+import assert from 'assert';
 import {
   InlineField,
   InlineFieldProps,
 } from '../private/InlineField/InlineField';
+import { RadioGroupContext } from '../RadioGroup/RadioGroupContext';
+import dedent from 'dedent';
 
 export interface RadioProps
   extends Omit<
     InlineFieldProps,
     'message' | 'reserveMessageSpace' | 'required'
   > {}
 
-const NamedRadio = forwardRef<HTMLInputElement, RadioProps>((props, ref) => (
-  <InlineField
-    {...props}
-    type="radio"
-    message={null}
-    reserveMessageSpace={false}
-    required={undefined}
-    ref={ref}
-  />
-));
+const NamedRadio = forwardRef<HTMLInputElement, RadioProps>((props, ref) => {
+  const radioListContext = useContext(RadioGroupContext);
+
+  if (process.env.NODE_ENV !== 'production') {
+    assert(
```

Comment:
This one should be outside of the env check so it runs in Playroom. Also, should be `radioGroupContext`.